app/src/main/res/values/strings.xml

```diff
@@ -436,6 +436,10 @@
     <string name="tabs_header_private_tabs_title">Private tabs</string>
     <!-- Content description (not visible, for screen readers etc.): Add tab button. Adds a news tab when pressed -->
     <string name="add_tab">Add tab</string>
+    <!-- Content description (not visible, for screen readers etc.): Add tab button. Adds a news tab when pressed -->
+    <string name="add_private_tab">Add private tab</string>
+    <!-- Content for FAB when looking at private tabs in the tab drawer -->
```

Comment:
Hi, can someone update this comment to explain what FAB stands for, if it's relevant for localizers?

We're starting to have a bit too many strings hold by this question
https://github.com/mozilla-l10n/android-l10n/pull/225/files